src/modules/modals/FileActionImport.vue

```diff
@@ -1,8 +1,7 @@
 <template>
-	<NcModal v-if="!importResults">
+	<NcDialog v-if="!importResults"
```

Comment:
The dialog when you import via file action never closes when you press the close button -- it always reappears. Probably due to how the modals are working vs. the dialogs. So maybe it would help to add something like 
```suggestion
	<NcDialog v-if="!importResults" @closing="closeImportDialog"
```

And further down in methods you can call `this.$emit('close')` like the others

Reply:
thanks